conf/production.ini

```diff
@@ -89,4 +89,9 @@ py-tracebacker = /tmp/via-traceback-
 #env = VIA_IGNORE_PREFIXES=https://hypothes.is/,https://qa.hypothes.is/,https://cdn.hypothes.is/
 
 mount = /=viahtml/wsgi.py
-uwsgi-socket = /tmp/viahtml-uwsgi.sock
\ No newline at end of file
+uwsgi-socket = /tmp/viahtml-uwsgi.sock
+
+stats =  0.0.0.0:3033
+stats-http = true
+stats-no-cores = true
+stats-minified = true
```

Comment:
Without `stats-http` the file is served as soon as you make a connection (no HTTP/1 headers etc), which is probably faster, but means doing some awful socket stuff.

These other options cut down the size of things you get back.